admin/app/conf/AdminConfiguration.scala

```diff
@@ -29,6 +29,7 @@ object AdminConfiguration {
 
   object fastly {
     lazy val key = configuration.getStringProperty("fastly.key").getOrElse(throw new RuntimeException("Fastly key not configured"))
+    lazy val serviceId = configuration.getStringProperty("fastly.serviceId").getOrElse(throw new RuntimeException("Fastly key not configured"))
```

Comment:
should be fastly.serviceId not nconfigured.  and I'm surprised we don't have a helper method to do this automatically correctly?